examples/fit_no_covariates.py

```diff
@@ -0,0 +1,268 @@
+"""
+This example constructs makes a test disease model, similar to
+diabetes, and sends that data to DismodAT.
+
+1.  The test model is constructed by specifying functions for
+    the primary rates, incidence, remission, excess mortality rate,
+    and total mortality. Then this is solved to get prevalence over time.
+
+2.  Then construct demographic observations of these rates. This means
+    averaging over the rates for given ages and times.
+
+3.  Load a subset of these observations into a DismodAT file and run
+    DismodAT on it. The commands to run are::
+
+        set option quasi_fixed false
+        set option ode_step_size 1
+        init
+        fit fixed
+        predict fit_var
+
+This example works in cohort time, so that rates don't change over
+years.
+"""
+from argparse import Namespace
+import itertools as it
+import logging
+from pathlib import Path
+import pickle
+from timeit import default_timer as timer
+
+import numpy as np
+import pandas as pd
+
+import cascade.input_data.db.bundle
+from cascade.testing_utilities import make_execution_context
+from cascade.dismod.db.metadata import IntegrandEnum, DensityEnum
+from cascade.core.context import ModelContext
+from cascade.dismod.serialize import write_to_file
+
+
+LOGGER = logging.getLogger("fit_no_covariates")
+
+
+def cached_bundle_load(context, bundle_id, tier_idx):
```

Comment:
This one is very cheesy. It could be a problem if they don't know it's happening.  I see an info-level message, though. If I set the rest of the messaging to debug, that will be OK.

Yes, I advocate theft with attribution. Not retribution, attribution.

Reply:
Not much to steal. It's just a usage pattern. But yes, let's do it. In a different PR.